proto/dymension/delayedack/rollapp_packet.proto

```diff
@@ -8,14 +8,22 @@ option go_package = "github.com/dymensionxyz/dymension/x/delayedack/types";
 
 message RollappPacket {
     ibc.core.channel.v1.Packet packet = 1;
+    bytes acknowledgement = 2;
+    bool isTimeout = 3;
     enum Status {
       PENDING = 0;
       ACCEPTED = 1;
       REJECTED = 2;
     }
-    Status status = 2;
-    uint64 ProofHeight = 3;
-    string error = 4;
-    bytes relayer = 5;
+    Status status = 4;
+    uint64 ProofHeight = 5;
+    string error = 6;
```

Comment:
not used

Reply:
Added usage for the 'error' field and fixed a bug where the methods weren't wrapped with `ApplyFuncIfNoError`